packages/module/patternfly-docs/content/extensions/chatbot/design-guidelines.md

```diff
@@ -222,22 +296,36 @@ There are a few display modes that users can choose when interacting with a Chat
 ![ChatBot in overlay mode.](./img/overlay.svg)
 </div>
 
-2. **Docked:** Anchors the ChatBot to the side of the page content. When docked, the ChatBot window is persistent, and cannot be toggled.
+2. **Docked:** Anchors the ChatBot to the side of the page, on top of the page content. When docked, the ChatBot window is persistent, and cannot be toggled. To minimize the ChatBot, users will need to return to overlay mode. 
```

Comment:
Technically, users can add a close button to the header. We just don't provide examples of this.